pkg/controller/datavolume/controller-base.go

```diff
@@ -388,6 +402,26 @@ func getSourceRefOp(log logr.Logger, dv *cdiv1.DataVolume, client client.Client)
 	}
 }
 
+func updatePendingDataVolumesGauge(log logr.Logger, dv *cdiv1.DataVolume, c client.Client) {
+	if cc.GetStorageClass(dv) != nil {
```

Comment:
a little confused about this;
I thought this func was about arbitrary DVs but you pass in a specific DV to it here,
and later perform the check on arbitrary DV `if cc.GetStorageClass(&dv) == nil {`

What am I missing?

Reply:
This func is called on any DV create/update/delete, but we are interested only on DVs `Pending` on default storage class, so if we are called form any other DV we can return immediately, as it won't change the deafult-SC-pending-DVs picture.